Mobile/MachineLearning/TextClassification/TextClassification/TextClassification/Models/TextClassification.cs

```diff
@@ -0,0 +1,181 @@
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ * http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+using System;
+using System.Collections.Generic;
+using System.Collections;
+using System.Text;
+using System.IO;
+using Tizen.MachineLearning.Inference;
+using Log = Tizen.Log;
+
+namespace TextClassification
+{
+    /// <summary>
+    /// Handle the input text
+    /// </summary>
+    public class HandleText
+    {
+        const string TAG = "TextClassification.Models";
+
+        private static string ResourcePath = Tizen.Applications.Application.Current.DirectoryInfo.Resource;
+        public string[] tokenized_label;
+        public string[] tokenized_vocab;
+        private Hashtable ht = new Hashtable();
+
+        /// <summary>
+        /// HandleText Constructor
+        /// </summary>
+        public HandleText()
+        {
+        }
+
+        /// <summary>
+        /// Split the input text and convert to float array.
+        /// </summary>
+        /// <param name="str"> user input string </param>
+        /// <returns> Converted float array from user input </returns>
+        public float[] SplitStr(string str)
+        {
+            float[] float_array = new float[256];
+            int cnt, len, start, unknown, pad, value;
+
+            start = Int32.Parse((string)ht["<START>"]);
+            pad = Int32.Parse((string)ht["<PAD>"]);
+            unknown = Int32.Parse((string)ht["<UNKNOWN>"]);
+
+            float_array[0] = (float)start;
+
+            /// Split words and convert to lowercase
+            string[] words = str.ToLower().Split(' ');
+            len = words.Length;
+
+            for (cnt = 1; cnt <= len; cnt++)
+            {
+                /// Check words and fill the float array with proper values
+                if (ht.ContainsKey(words[cnt - 1]))
+                {
+                    value = Int32.Parse((string)ht[words[cnt - 1]]);
+                    float_array[cnt] = (float)value;
+                }
+                else
+                {
+                    Log.Debug(TAG, "Given word is not contained.");
+                    float_array[cnt] = (float)unknown;
+                }
+            }
+
+            /// Remainers are filled with 0
+            while (cnt < 256)
+            {
+                float_array[cnt++] = (float)pad;
+            }
+
+            return float_array;
+        }
+
+        /// <summary>
+        /// Load files and make hash table
+        /// </summary>
+        public void Init_model()
+        {
+            string label = ResourcePath + "models/labels.txt";
```

Comment:
```suggestion
            string label = Path.Combine(ResourcePath, "models/labels.txt");
```
https://docs.microsoft.com/en-us/dotnet/api/system.io.path.combine?view=netframework-4.8#System_IO_Path_Combine_System_String_System_String_

Reply:
Thank you. I will change it.